draft-ietf-quic-recovery.md

```diff
@@ -538,9 +539,9 @@ max_ack_delay:
   RTT sample less than min_rtt.
 
 reordering_threshold:
-: The largest delta between the largest acked
-  retransmittable packet and a packet containing retransmittable frames before
-  it's declared lost.
+: The largest packet number gap between the largest acked
+  congestion controlled packet and smaller unacknowledged congestion
```

Comment:
This is the first reference you have to `congestion controlled packet`. You haven't explicitly defined what this means. I assume it is just a packet that contains at least 1 congestion controlled frame, but I think it would be worth explicitly stating that above.

Reply:
Thanks, done above.